columnflow/columnar_util.py

```diff
@@ -2437,6 +2462,19 @@ def __call__(
 
         return result
 
+    def get_min_chunk_size(self) -> int | None:
```

Comment:
probably irrelevant but could this be a cached property?

```suggestion
    @functools.cached_property
    def get_min_chunk_size(self) -> int | None:
```

Reply:
ah no I though this is called in a loop, but it should in any case only be called once per task.